packages/proto-build/src/main.rs

```diff
@@ -45,17 +44,14 @@ pub fn generate(version_tags: &HashMap<String, String>) {
     let ibc_go_version = version_tags
         .get("IBC_GO_VERSION")
         .expect("IBC_GO_VERSION is not set");
-    let ics23_version = version_tags
-        .get("ICS23_VERSION")
-        .expect("ICS23_VERSION is not set");
-
-    if args.iter().any(|arg| arg == "--update-deps") {
-        git::update_submodule(FINSCHIA_SDK_DIR, &finschia_sdk_version);
-        git::update_submodule(WASMD_DIR, &wasmd_version);
-        git::update_submodule(COMETBFT_DIR, &tendermint_version);
-        git::update_submodule(IBC_GO_DIR, &ibc_go_version);
-        git::update_submodule(ICS23_DIR, &ics23_version);
-    }
+
+    // cosmos/ics23 have not supported yet in Finschia. So it is fixed tag as like osmosis.
+    let ics23_version = "rust/v0.10.0";
+    git::checkout_submodule(FINSCHIA_SDK_DIR, &finschia_sdk_version);
+    git::checkout_submodule(WASMD_DIR, &wasmd_version);
+    git::checkout_submodule(COMETBFT_DIR, &tendermint_version);
+    git::checkout_submodule(IBC_GO_DIR, &ibc_go_version);
+    git::checkout_submodule(ICS23_DIR, &ics23_version);
```

Comment:
please check

Reply:
"ics23" will be supported once IBC support is available.
https://github.com/Finschia/finschia-wasm/issues/10